Newsletters/Yearn Finance Newsletter #48/Yearn Finance Newsletter #48.md

```diff
@@ -1,135 +1,133 @@
+El reino de multi blockchain ha hecho el llamado. Y Yearn ha acudido.
 
-https://beta.yearn.finance
+Hoy, nos convertimos en multi blockchain con el lanzamiento del Iron Bank Fantom y las primeras vaults en Fantom en https://beta.yearn.finance
 
-You may ask, “Why multichain now?”. This is because Yearn requires specific infrastructure to safely and efficiently function, both external (Gnosis safe, DEX liquidity, block explorer) and internal (frontend and backend).
+Te preguntarás, ¿por qué multi blockchain ahora?. Es porque Yearn requiere una infraestructura específica para funcionar de forma eficiente, tanto de forma externa (Gnosis safe, liquidez en DEX, explorador de bloques) e internamente (frontend y backend).
 
-Our v2 vault codebase has hardened over the past few months, and the new beta website is a vast improvement, allowing us to switch chains relatively easily, something we simply could not have done in the past.
+La base de código de nuestra vault V2 se ha fortalecido en los pasados meses, y la nueva web versión beta ha sido una mejora importante, permitiéndonos cambiar de blockchain con relativa facilidad, algo que implemente no podíamos hacer en el pasado.
 
-Fantom is fast, simple to use, and easy to bridge to thanks to Anyswap. Additionally, it is well-supported by our development tools and by our Iron Bank partners at CREAM Finance. Check out the article on the Fantom Iron Bank [here](https://medium.com/cream-finance/c-r-e-a-m-dd4982004bb9).  
+Fantom es rápido, simple de usar y permite utilizar puentes fácilmente, gracias a Anyswap. Adicionalmente, está muy bien soportada por nuestras herramientas de desarrollo y por nuestros colaboradores del Iron Bank en CREAM finance. Echa un vistazo al artículo sobre el Iron Bank en Fantom [Aquí](https://medium.com/cream-finance/c-r-e-a-m-dd4982004bb9). 
   
-And it doesn't hurt that Andre Cronje is a big fan.
+Desde luego no hace daño tampoco que Andre Cronje sea un gran fan.
 
-The first vaults on Fantom are: yvWFTM, yvUSDC, yvDAI, and yvMIM. We’ve launched support for a native Fantom Iron Bank as well. Head to [beta.yearn.finance](https://beta.yearn.finance) to check them out for yourself.
+Las primeras vaults en Fantom son: yvWFTM, yvDAI y yvMIM. Lanzamos también soporte nativo para el Iron Bank de Fantom. Puedes verlo por ti mismo en [beta.yearn.finance](https://beta.yearn.finance).
 
-So to answer your questions of when Arbitrum, Optimism, Polygon, Avalanche, and others are arriving, there will definitely be more multichain fun coming soon. Ultimately, we’d love to bring Yearn to any chain where we can safely operate and simplify the process of earning yield for users and partners.
+Para responder a tu pregunta, Arbitrum, Optimism, Polygon, Avalanche y otras están disponibles pronto. Finalmente nos encantaría llevar a Yearn a cualquier otra blockchain donde pueda operar con seguridad y así simplificar los procesos de obtención de rendimiento para sus los usuarios y colaboradores.
```

Comment:
AS-IS: Para responder a tu pregunta, Arbitrum, Optimism, Polygon, Avalanche y otras están disponibles pronto. Finalmente nos encantaría llevar a Yearn a cualquier otra blockchain donde pueda operar con seguridad y así simplificar los procesos de obtención de rendimiento para sus los usuarios y colaboradores.

TO-BE: Para responder a tu pregunta, Arbitrum, Optimism, Polygon, Avalanche y otras estarán disponibles pronto. Finalmente nos encantaría llevar a Yearn a cualquier otra blockchain donde pueda operar con seguridad y así simplificar los procesos de obtención de rendimiento para sus los usuarios y colaboradores.